src/Card/CardSection/index.js

```diff
@@ -90,12 +91,34 @@ class CardSection extends React.Component<any, Props> {
     }
   };
 
+  handleKeyPress = (ev: SyntheticKeyboardEvent<HTMLInputElement>) => {
+    if (ev.key === "Enter" || ev.key === " ") {
```

Comment:
In this case would be better to use `ev.code` it returns `Enter` or `Space`

Reply:
I tried `.code` before, not sure why but I'm getting `undefined` not sure what am I missing.